server/email_verification.go

```diff
@@ -236,6 +328,12 @@ func handleEmailVerifyFunc(verifiedTpl *template.Template, issuer url.URL, keysF
 			}
 			return
 		}
+
+		qr := cbURL.Query()
+		qr.Set("code", code)
+		qr.Set("state", ses.ClientState)
```

Comment:
The email invitation API call would require code, scope, and nonce parameters. Basically all the parameters the auth endpoint takes on initial request.